zio-http-example/src/main/scala/example/ClientServer.scala

```diff
@@ -17,6 +17,7 @@ object ClientServer extends ZIOAppDefault {
 
   val run = {
     val clientLayers = ChannelFactory.auto ++ EventLoopGroup.auto()
-    Server.start(8080, app).provideLayer(clientLayers).exitCode
+    Server.serve(app).provide(Server.default ++ clientLayers)
```

Comment:
In all the examples, try to let ZIO do the wiring for you: `.provide(Server.default, clientLayers)`.

Reply:
Done.